src/improvements/tasks/eth/016-worldchain-l1pao-transfers/VALIDATION.md

```diff
@@ -0,0 +1,117 @@
+
+- **Key:**          `0x0000000000000000000000000000000000000000000000000000000000000033`
+  - **Decoded Kind:** `address`
+  - **Before:** [`0xA4fB12D15Eb85dc9284a7df0AdBC8B696EdbbF1d`](https://github.com/ethereum-optimism/superchain-registry/blob/d82a61168fd1d7ef522ed8e213ce23c853031495/superchain/configs/mainnet/worldchain.toml#L43)
+  - **After:** [`0x6aACd82e5D5A41aC508D456a151Ec53d2b1Fd7ab`](https://etherscan.io/address/0x6aACd82e5D5A41aC508D456a151Ec53d2b1Fd7ab)
+  - **Summary:** DisputeGameFactory owner update to new 2-of-2 OP Foundation-Worldchain Safe
+  - **Detail:** Verify the slot `0x0000000000000000000000000000000000000000000000000000000000000033` is correct by running the following command and observing that the output is the same as the `Before` value:
+    
+    ```bash
+    cast storage 0x069c4c579671f8c120b1327a73217d01ea2ec5ea 0x0000000000000000000000000000000000000000000000000000000000000033 --rpc-url mainnet
+    # returns 0x000000000000000000000000a4fb12d15eb85dc9284a7df0adbc8b696edbbf1d
+    cast call 0x069c4c579671f8c120b1327a73217d01ea2ec5ea "owner()(address)" --rpc-url mainnet
+    # returns 0xA4fB12D15Eb85dc9284a7df0AdBC8B696EdbbF1d
+    ```
+  ---
+  
+### [`0x4e6de8b4c2d5ad6c603648f78311a21558d37a53`](https://etherscan.io/address/0x4E6dE8B4c2D5aD6c603648f78311a21558D37A53) (DelayedWETH) - Chain ID: 480
+
+- **Key:**          `0x0000000000000000000000000000000000000000000000000000000000000033`
+  - **Decoded Kind:** `address`
+  - **Before:** [`0xB2aa0C2C4fD6BFCBF699d4c787CD6Cc0dC461a9d`](https://github.com/ethereum-optimism/superchain-registry/blob/d82a61168fd1d7ef522ed8e213ce23c853031495/superchain/configs/mainnet/worldchain.toml#L42)
+  - **After:** [`0x6aACd82e5D5A41aC508D456a151Ec53d2b1Fd7ab`](https://etherscan.io/address/0x6aACd82e5D5A41aC508D456a151Ec53d2b1Fd7ab)
+  - **Summary:** Permissioned DelayedWETH owner update to new 2-of-2 OP Foundation-Worldchain Safe
+  - **Detail:** Verify the slot `0x0000000000000000000000000000000000000000000000000000000000000033` is correct by running the following command and observing that the output is the same as the `Before` value:
+    
+    ```bash
+    cast storage 0x4e6de8b4c2d5ad6c603648f78311a21558d37a53 0x0000000000000000000000000000000000000000000000000000000000000033 --rpc-url mainnet
+    # returns 0x000000000000000000000000b2aa0c2c4fd6bfcbf699d4c787cd6cc0dc461a9d
+    cast call 0x4e6de8b4c2d5ad6c603648f78311a21558d37a53 "owner()(address)" --rpc-url mainnet
+    # returns 0xB2aa0C2C4fD6BFCBF699d4c787CD6Cc0dC461a9d
+    ```
+  ---
+  
+### [`0xa4fb12d15eb85dc9284a7df0adbc8b696edbbf1d`](https://github.com/ethereum-optimism/superchain-registry/blob/d82a61168fd1d7ef522ed8e213ce23c853031495/superchain/configs/mainnet/worldchain.toml#L43C14-L43C15) (ProxyAdminOwner (GnosisSafe)) - Chain ID: 480
+
+- **Key:**          `0x0000000000000000000000000000000000000000000000000000000000000005`
+  - **Decoded Kind:** `uint256`
+  - **Before:** `29`
```

Comment:
Useful for signers if you add the hex representation (like the sepolia PR) for both nonces.